resources/chefdk/msi/source.wxs.erb

```diff
@@ -62,19 +65,30 @@
           </Directory>
         </Directory>
       </Directory>
+      
+      <Directory Id="ProgramMenuFolder">
+        <Component Id="StartMenuShortcut" Guid="{6DD3FFF3-E009-40E7-B1C2-EF606B941CCD}">
+            <Shortcut Id="StartMenuShortcutDef" 
```

Comment:
It will default to the directory of the Component (since we don't really care where it runs, we let it do that).